component/src/main/java/org/cbioportal/genome_nexus/component/annotation/VariantClassificationResolver.java

```diff
@@ -186,9 +186,13 @@ private static Map<String, String> initVariantMap()
         variantMap.put("missense_variant",              "Missense_Mutation");
         variantMap.put("protein_altering_variant",      "Missense_Mutation"); // Not always correct, resolveVariantClassification handles the exceptions
         variantMap.put("coding_sequence_variant",       "Missense_Mutation"); // Not always correct, resolveVariantClassification handles the exceptions
+        variantMap.put("coding_transcript_variant",       "Missense_Mutation"); // Not always correct, resolveVariantClassification handles the exceptions
         variantMap.put("conservative_missense_variant", "Missense_Mutation");
         variantMap.put("rare_amino_acid_variant",       "Missense_Mutation");
         variantMap.put("transcript_amplification",      "Intron");
+        variantMap.put("feature_elongation",      "Feature_Mutation");
+        variantMap.put("feature_truncation",      "Feature_Mutation");
+        variantMap.put("sequence_variant",      "Sequence_Mutation");
```

Comment:
Need advice on what variant classification these should be. I put "Feature_Mutation" and "Sequence_Mutation" for now but these are not official names 🙂

Reply:
We decide to keep original name as the classification